### ISSUE TITLE
Dockerfile to build themes

### ISSUE THREAD

**Issue by MoriTanosuke** [geokapp/arc-variants#2]
I created a simple `Dockerfile` to build themes on my machine, because I don't want to install all the dependencies into my system.

Add the following `Dockerfile` to your local directory:

    FROM ubuntu:16.10
    ENV VARIANT=blue-hidpi
    RUN apt-get update -y && apt-get install -y autoconf automake pkg-config libgtk-3-dev git npm nodejs optipng inkscape
    RUN ln -s /usr/bin/nodejs /usr/bin/node

Build the image:

    docker build -t arc-variants .

Start a temporary container, will be removed when everything is done:

    docker run --rm -it -v $(pwd):/src -w /src arc-variants

Then build the theme in the temporary container:

    ./arc-variants --install=blue-hidpi && cp -r /usr/share/themes/Arc* . && exit

Now you should have the theme directories in the local directory on your host. Copy the files into `/usr/share/themes` and switch to your ARC theme.

If you don't want a `Dockerfile` in your repository, I'll just post it on my blog to not loose it. :scroll:

**Comment by geokapp** [comment]
Thanks for this! [Done](https://github.com/geokapp/arc-variants/commit/f7f3a27ce86ae94a18a2c64fc1fd6e10a5c467bd) :)

**Comment by StreetStrider** [comment]
@geokapp @MoriTanosuke I've tried to render theme inside container, got one error in process.
`who` returns empty result (and `$USER` too), and [this line](https://github.com/geokapp/arc-variants/blob/a025b9b44d1bcdec92b425a86ba58103679971d6/arc-variants#L226) not working as expected. I've replaced the whole `$(who ...)` stuff with `root` literal and from that render works flawlessly. What's with the behavior of `who` according to docker?
